aten/src/ATen/native/native_functions.yaml

```diff
@@ -12879,10 +12881,16 @@
     CompositeExplicitAutogradNonFunctional: _reshape_alias_copy_symint
   tags: view_copy
 
-- func: select_copy.int(Tensor self, int dim, int index) -> Tensor
+- func: select_copy.Dimname(Tensor self, Dimname dim, SymInt index) -> Tensor
+  variants: function
+  dispatch:
+    CompositeExplicitAutogradNonFunctional: select_copy_symint
+  tags: view_copy
```

Comment:
After adding `SymInt` to `select.int`, couldn't compile without adding this copy variant. Can we perhaps just remove the `select.Dimname` overload? Not sure, but IIRC `Dimname`s are experimental and will be removed later anyway, no? Or is that BC-breaking?

Reply:
It's bc breaking but we do want to remove them. I noticed you symintified a dimname, maybe undoing that would help